src/tests/mocks/panzoom.ts

```
@@ -0,0 +1,53 @@
+
+  const mock = {
+    getTransform: () => ({ ...transform }),
+    zoomAbs: vi.fn((x: number, y: number, scale: number) => {
+      const oldScale = transform.scale || 1;
+      const scaleRatio = scale / oldScale;
+      transform = {
+        x: x - (x - transform.x) * scaleRatio,
+        y: y - (y - transform.y) * scaleRatio,
+        scale,
+      };
+      listeners["zoom"]?.forEach((cb) => cb());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)